requirements.txt

```diff
@@ -1,11 +1,11 @@
-Django==2.0
+Django~=2.2
 celery==4.1.1
 django-bootstrap-form==3.2.1
 oauthlib==2.0.1
 requests==2.20.0
 git+https://github.com/Aalto-LeTech/django-lti-login.git@master#egg=django-lti-login
 cachetools==2.0.1
-git+https://github.com/raphendyr/raphendyr-django-essentials.git@1.2.1
+git+https://github.com/raphendyr/raphendyr-django-essentials.git@1.3.0
```

Comment:
This could use the pip package, instead of git, thus you could replace this with `raphendyr-django-essentials ~= 1.3.0`.

(If git is used, then the correct url would be `git+https://github.com/raphendyr/raphendyr-django-essentials.git@1.3.0#egg=raphendyr-django-essentials==1.3.0` to ensure the package is updated)